packages/components/src/VirtualizedList/CellIconText/CellIconText.scss

```diff
@@ -1,13 +1,11 @@
-.tc-virtualizedlist-cell-icon-text {
+.tc-icon-text {
 	display: flex;
-	:global(.tc-svg-icon) {
-		margin-right: $padding-small;
-	}
+}
 
-	&-label {
-		min-width: 0;
-		text-overflow: ellipsis;
-		overflow: hidden;
-		white-space: nowrap;
-	}
+.label {
```

Comment:
If you still want css module to output you a classname with context (~`tc-icon-text__label__hash`), you can nest `.label` in `.tc-icon-text`.
I approve anyway, as the label style won't leak in the current state, it's just a matter of your taste :)

Reply:
I'll give another try on that